specification/sbi_atee.adoc

```diff
@@ -889,6 +889,31 @@ enum Exception {
 };
 -------
 
+[#sbi_tee_host_tvm_fence]
+=== Function: TEE Host Initiate TVM Fence (FID #15)
+[source, C]
+-----
+struct sbiret sbi_tee_host_tvm_fence(unsigned long tvm_guest_id);
+-----
+Initiates a TLB invalidation sequence for all pages that have been invalidated in the
+given TVM's address space since the previous call to `sbi_tee_host_tvm_fence()`. The TLB
+invalidation sequence is completed when all vCPUs in the TVM that were running prior to
+the call to `sbi_tee_host_tvm_fence()` have taken a trap into the TSM, which the host can
+cause by IPI'ing the physical CPUs on which the TVM's vCPUs are running. An error is
```

Comment:
/s/IPI'ing/sending IPIs to

Reply:
Fixed. My bad, leaked from salus comment documentation.